src/constants/ArtifactModifiers.js

```diff
@@ -0,0 +1,30 @@
+const ArtifactModifiers = {
+  "4pc_noblesse": {
+    label: "4pc Noblesse",
+    field: "4pc_noblesse",
+    burst: 1.2,
```

Comment:
think about how this is going into the damage bonus multiplier, which takes in percentages.
Make this 20% instead